src/js/model.js

```diff
@@ -34,57 +37,68 @@ export const mixerList = [
 
 // 3D model
 const Model = function (wrapper, canvas) {
-    const useWebGLRenderer = this.canUseWebGLRenderer();
+    // Configure model detail level (1-10, where 1 is lowest detail and 10 is highest)
+    this.detailTolerance = 10; // Default value, can be modified
+
+    this.useWebGLRenderer = this.canUseWebGLRenderer();
 
     this.wrapper = wrapper;
     this.canvas = canvas;
 
-    if (useWebGLRenderer) {
-        this.renderer = new THREE.WebGLRenderer({ canvas: this.canvas[0], alpha: true, antialias: true });
+    if (this.useWebGLRenderer) {
+        this.renderer = new THREE.WebGLRenderer({
+            canvas: this.canvas[0],
+            alpha: true,
+            antialias: true, // Disable antialiasing for performance
+        });
     } else {
-        this.renderer = new THREE.CanvasRenderer({ canvas: this.canvas[0], alpha: true });
+        console.log("Starting in low performance rendering mode");
+        this.renderer = new CanvasRenderer({
+            canvas: this.canvas[0],
+            alpha: true,
+        });
     }
 
-    // initialize render size for current canvas size
     this.renderer.setSize(this.wrapper.width(), this.wrapper.height());
 
     // load the model including materials
-    let model_file = useWebGLRenderer ? mixerList[FC.MIXER_CONFIG.mixer - 1].model : "fallback";
-
+    //let model_file = this.useWebGLRenderer ? mixerList[FC.MIXER_CONFIG.mixer - 1].model : "fallback";
```

Comment:
fallback is needed for models without svg or other format.

Reply:
Fixed, but I have noticed that there is already a bug present in master branch. When fallback model is active there is no model presented. This screenshot is from master branch. 

<img width="961" alt="image" src="https://github.com/user-attachments/assets/3b4a82ac-d7f9-4370-a5b7-2ab205cdf145" />

Reply:
Custom on master

![image](https://github.com/user-attachments/assets/e83972c4-8fb6-41f1-a600-fa6dc5cdb40d)

Reply:
Same using the deployment preview

![image](https://github.com/user-attachments/assets/bf1645e7-bc1e-4949-9c3a-7d12a7e6e4b2)

Reply:
And with legacy options enabled

![image](https://github.com/user-attachments/assets/5cefb3bd-b753-479d-b790-94c52536ab6f)

Reply:
Wow! Ok, that is  great then, I will check to my local setup, for me behaves quite different locally. Thanks for taking a look.